selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/workspaces/WorkspaceOverview.java

```diff
@@ -88,6 +90,11 @@ public Boolean isWorkspaceNameErrorMessageEquals(String message) {
     return errorMessages.getText().equals(message);
   }
 
+  public void waitWorkspaceNameIsValid() {
```

Comment:
Method name is not clear enough, IMHO.
May we name it as: _checkOnWorkspaceNameErrorAbsence()_?